x-pack/plugin/sql/qa/src/main/java/org/elasticsearch/xpack/sql/qa/jdbc/JdbcAssert.java

```diff
@@ -34,6 +38,15 @@
  * Utility class for doing JUnit-style asserts over JDBC.
  */
 public class JdbcAssert {
+
+    private static final IntObjectHashMap<DataType> SQL_TO_TYPE = new IntObjectHashMap<>();
+
+    static {
+        for (DataType type : DataType.values()) {
+            SQL_TO_TYPE.putIfAbsent(type.getVendorTypeNumber().intValue(), type);
```

Comment:
should we check and fail on duplicates?

Reply:
no as there will be duplicates. There's no 1 to 1 mapping between our types and SQL, many get mapped to `VARCHAR` for example. This table adds the obvious one - `VARCHAR` will be mapped to `KEYWORD` and not `IP`.
Of course this means the enum declaration order is important but since we don't want to change it anyway, I think that's good enough.